graphviz2drawio/models/SVG.py

```
@@ -26,6 +51,19 @@ def get_text(g: Element) -> str | None:
         return text_el.text
     return None
 
+def get_d(g: Element) -> str | None:
+    polygon_str = ET.tostring(g, encoding='unicode')
```

Ruff (0.8.0)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)